giga_connectome/data/methods/template.jinja

```diff
@@ -0,0 +1,28 @@
+These results were generated with
+giga_connectome (version {{ data.version }}, https://giga-connectome.readthedocs.io/en/latest/)
+using Nilearn (version {{ data.nilearn_version }}, RRID:SCR_001362)
+and TemplateFlow (version {{ data.templateflow_version }}).
+
+The following steps were followed.
+
+1. Created subject specific grey matter mask in MNI space ({{ data.mni_space }}).
+
+2. Sampled the {{ data.atlas }} atlas to the space of subject specific grey matter mask in MNI space.
+
+3. Calculated the conjunction of the customised grey matter mask and resampled atlas to find valid parcels.
+
+4. Used the new input specific grey matter mask and atlas to extract time series and connectomes for each subject.
+   The time series data were denoised as follow:
+
+    - Time series extractions through label or map maskers are performed on the denoised nifti file.
+    - Denoising steps were performed on the voxel level:
+        - spatial smoothing (FWHM: {{ data.smoothing_fwhm }} mm)
+        - detrending, only if high pass filter was not applied through confounds
+        - regressing out confounds (using a {{ data.strategy }} strategy)
+        - standardized (using {{ data.standardize }})
+    - Extracted time series from atlas
+    - Computed correlation matrix
+
+5. Calculate intranetwork correlation of each parcel. The values replace the diagonal of the connectomes.
```

Comment:
this step is optional; and I feel there's a need to flesh out the details of this thing

Reply:
OK I will adapt the template to only include it when people asked for it

Reply:
this is done: some section of the methods section will only be added if the user passes certain parameters